ChangeLog

```diff
@@ -1,6 +1,6 @@
-* Mon Oct 19 2020 Chris Lumens <clumens@redhat.com> Pacemaker-2.0.5
-- Changesets: 497
-- Diff: 285 files changed, 22704 insertions(+), 14458 deletions(-)
+* Tue Oct 27 2020 Chris Lumens <clumens@redhat.com> Pacemaker-2.0.5
```

Comment:
I missed this the last time ... this should have -rc2 in it

Usually we do separate entries for rc1/rc2/etc. then combine them at the end, but I don't see anything wrong with updating it as you go